dbms/src/Storages/Transaction/RegionPersister.cpp

```diff
@@ -246,8 +247,8 @@ RegionMap RegionPersister::restore(PathPool & path_pool, const TiFlashRaftProxyH
                     provider,
                     global_context.getPSBackgroundPool());
                 page_storage_v2->restore();
-                page_writer = std::make_shared<PageWriter>(global_run_mode, page_storage_v2, /*storage_v3_*/ nullptr);
-                page_reader = std::make_shared<PageReader>(global_run_mode, ns_id, page_storage_v2, /*storage_v3_*/ nullptr, /*readlimiter*/ global_context.getReadLimiter());
+                page_writer = std::make_shared<PageWriter>(run_mode, StorageType::KVStore, page_storage_v2, /*storage_v3_*/ nullptr, /*uni_ps_*/ nullptr);
+                page_reader = std::make_shared<PageReader>(run_mode, StorageType::KVStore, ns_id, page_storage_v2, /*storage_v3_*/ nullptr, /*uni_ps_*/ nullptr, /*readlimiter*/ global_context.getReadLimiter());
             }
             else
             {
```

Comment:
https://github.com/pingcap/tiflash/blob/6311b79b3a1946d4e255e163c31fb671a88265a2/dbms/src/Storages/Transaction/RegionPersister.cpp#L253-L262

Since PS V3 is enabled by default since v6.5 and the data is compatible for reading by PS V2. I think we can remove the legacy `stable_page_storage` codes to make RegionPersister more clear

Reply:
Removed it in https://github.com/pingcap/tiflash/pull/6781.